proto/desmos/subspaces/v1/query.proto

```diff
@@ -0,0 +1,150 @@
+syntax = "proto3";
+package desmos.subspaces.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "desmos/subspaces/v1/models.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v2/x/subspaces/types";
+
+// Query defines the gRPC querier service
+service Query {
+
+  // Subspaces queries all the subspaces inside Desmos
+  rpc Subspaces(QuerySubspacesRequest) returns (QuerySubspacesResponse) {
+    option (google.api.http).get = "/desmos/subspaces/v1/subspaces";
+  }
+
+  // Subspace queries all the information about the subspace with the given id
+  rpc Subspace(QuerySubspaceRequest) returns (QuerySubspaceResponse) {
+    option (google.api.http).get =
+        "/desmos/subspaces/v1/subspaces/{subspace_id}";
+  }
+
+  // UserGroups queries all the groups that are present inside the subspace with
+  // the given id
+  rpc UserGroups(QueryUserGroupsRequest) returns (QueryUserGroupsResponse) {
+    option (google.api.http).get = "/desmos/subspaces/v1/groups/{subspace_id}";
```

Comment:
As a rest api, should it follow the naming guide to be "/desmos/subspaces/v1/subspaces/{subspace_id}/groups"?